sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # Release History
 
-## 7.1.2 (Unreleased)
+## 7.2.0 (Unreleased)
 
+**New Features**
+
+* `ServiceBusAdministrationClient.update_*` methods now accept keyword arguments to override the properties specified in the model instance.
+
+**Bug Fixes**
 
+* Fixed a bug that `update_queue` and `update_subscription` mutating the properties `forward_to` and `forward_dead_lettered_messages_to` of the model instance when those properties are entities instead of full paths.
```

Comment:
small nit, wording:
```suggestion
* Fixed a bug where `update_queue` and `update_subscription` methods were mutating the properties `forward_to` and `forward_dead_lettered_messages_to` of the model instance when those properties are entities instead of full paths.
```